includes/roslyn-installation.md

```diff
@@ -0,0 +1,29 @@
+## Installation instructions 
+
+There are two different ways to find the **.NET Compiler Platform SDK** in the **Visual Studio Installer**:
+
+Install using the **Workloads** view:
```

Comment:
I think I'd do this as level-3 head, and change the "You can also install using..." to "Install using the Individual Components tab" as a level-3 head. Otherwise, the structure of this section seems unclear.